src/autobridgecalculator.js

```diff
@@ -46,7 +46,27 @@ AutobridgeCalculator.prototype.calculate = function() {
 
   this.clearOwnerFundsLeftover();
 
+  this._calculateInternal(legOnePointer, legTwoPointer, offersAutobridged,
+    callback);
+};
+
+AutobridgeCalculator.prototype._calculateInternal = function(
+  legOnePointer, legTwoPointer, offersAutobridged, callback = function() {}
+) {
+
+  const startTime = Date.now();
+
   while (this.legOneOffers[legOnePointer] && this.legTwoOffers[legTwoPointer]) {
+    const lasted = (Date.now() - startTime);
```

Comment:
Purpose is to split processing into chunks, so users browser can do something other than calculate offers. I'm using time instead of just splitting into predefined number of steps because user's device can have different speed.
Also, I've set timeout to 33ms so it slightly bigger than processing time - 30 - because RT client has two book open simultaneously, so two AutobridgeCalculator can be run at the same time, so I'm trying to give them change to work together  and not to burn users cpu.